client/etcd/options.go

```diff
@@ -35,14 +35,14 @@ type Options interface {
 	Zone() string
 	SetZone(z string) Options
 
-	ServiceInitTimeout() time.Duration
-	SetServiceInitTimeout(timeout time.Duration) Options
+	Service() string
```

Comment:
spoke to Chao offline, Service here is used as an identifier for the filename generation and metrics; doesn't mean the same thing as Name from ServiceID.